txcache/transactionsMaxHeap.go

```diff
@@ -0,0 +1,27 @@
+package txcache
+
+type TransactionsMaxHeap []*TransactionsHeapItem
+
+func (maxHeap TransactionsMaxHeap) Len() int { return len(maxHeap) }
```

Comment:
Good question!

`TransactionsMaxHeap` was a slice, and the functions that did not change it received it "by value" (tiny struct). While other functions - such as push and pop - use pointer receivers because they modify the slice's length.

However, the refactored `TransactionsHeap` is not a slice anymore, but a struct. Now using pointer as function receiver.

Reply:
ok